src/main/app/claims/models/paymentDeclaration.ts

```diff
@@ -1,10 +1,11 @@
 export class PaymentDeclaration {
 
-  constructor (public paidDate?: string, public explanation?: string) {}
+  constructor (public paidDate?: string, public paidAmount?: number, public explanation?: string) {}
```

Comment:
just a thought - should we keep whats optional and whats not optional consistent between frontend and backend?

Reply:
Yes. What is the disparity?

Reply:
![image](https://user-images.githubusercontent.com/12914259/52856074-51565080-311b-11e9-930d-174272c3a773.png)

Reply:
`@NotNull
    @DateNotInTheFuture
    private final LocalDate paidDate;

    @NotBlank
    @Size(max = 99000)
    private final String explanation;`

above in master

Reply:
I have another PR for updates to the claim-store.
https://github.com/hmcts/cmc-claim-store/pull/742

Reply:
Yep, they're optional in both.

Reply:
in frontend i will be able to do `new PaymentDeclaration()` and backend will throw an error because of existing validations. My point was, shouldn't that be consistent? I don't have complete context of the change you are making, so it doesn't look right to me that we depend validation errors completely on backend when we could catch them at frontend. I understand you are only adding a property here, so you don't have to make this change as part of this PR.... 

@sabahirfan - any thoughts on this?

Reply:
There is validation on the front-end, it's just a bit convoluted because of the different models. Validation of these data is done in `src/main/features/response/form/models/howMuchHaveYouPaid.ts` and this file is just used for data conversion to the back-end model

Reply:
thanks for explaining, happy with that